bounce/server/api/users.py

```diff
@@ -72,13 +75,16 @@ class UserEndpoint(Endpoint):
             #  that we can use to securely
             # verify their password when they log in
             secret = util.hash_password(body['new_password'])
+        elif body.get('bio'):
+            bio = body['bio']
         # Update the user
         updated_user = user.update(
             self.server.db_session,
             username,
             secret=secret,
             full_name=body.get('full_name', None),
-            email=email)
+            email=email,
+	        bio=bio)
```

Comment:
You can just do `bio=body.get('bio', None)` here, then you don't have to add any of the code above.